### ISSUE TITLE
Look into missing alt text on Invalid src prop Image example

### ISSUE THREAD

**Issue by bjankord** [cerner/terra-core#2306]
# Issue Description
While run axe-core automated tests on terra-ui.com, it reports that [this page](https://engineering.cerner.com/terra-ui/#/components/terra-image/image/image) has a missing alt text attribute on one of the images.

The Invalid src prop example has an img element with the following alt text. `alt=" "`
<img width="552" alt="Screen Shot 2019-03-26 at 3 09 47 PM" src="https://user-images.githubusercontent.com/633148/55030032-79b65200-4fd9-11e9-9cf5-cac3e88e46b9.png">

This is odd as [the src code has the alt attribute filled out for both the invalid src image and the placeholder image](https://github.com/cerner/terra-core/blob/master/packages/terra-image/src/terra-dev-site/doc/example/ImageDefault.jsx#L27). Need to investigate and see why the alt text in this example is being rendered as a space character instead of the alt attribute provide on the `<Image />` component.

**Comment by yuderekyu** [comment]
The 2nd `alt` is outside the placeholder, it's applied to the outer image. The placeholder `alt` then defaults to a blank string
